README.md

```diff
@@ -41,21 +41,29 @@ The pipeline is built using [Nextflow](https://www.nextflow.io), a workflow tool
     nextflow run nf-core/mag -profile <docker/singularity/podman/conda/institute> --input '*_R{1,2}.fastq.gz'
```

Comment:
This example could be updated to show also the use of the manifest file

Reply:
True, also citations for all tools used are missing from the readme. I opened an issue for this #156